kernels/volk/volk_32f_log2_32f.h

```diff
@@ -92,29 +94,23 @@
 #ifndef INCLUDED_volk_32f_log2_32f_a_H
 #define INCLUDED_volk_32f_log2_32f_a_H
 
-#include <stdio.h>
-#include <stdlib.h>
 #include <inttypes.h>
 #include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
 
 #define LOG_POLY_DEGREE 6
 
-// +-Inf -> +-127.0f in order to match the behaviour of the SIMD kernels
-static inline float log2f_non_ieee(float f) {
-  float const result = log2f(f);
-  return isinf(result) ? copysignf(127.0f, result) : result;
-}
-
 #ifdef LV_HAVE_GENERIC
 
-static inline void
-volk_32f_log2_32f_generic(float* bVector, const float* aVector, unsigned int num_points)
-{
-  float* bPtr = bVector;
-  const float* aPtr = aVector;
+static inline void volk_32f_log2_32f_generic(float *bVector,
+                                             const float *aVector,
+                                             unsigned int num_points) {
+  float *bPtr = bVector;
```

Comment:
i don't know clang-format; like really at all. can we change it to do `float* foo;` rather than `float *foo;`? My preference is keeping the whole type all together.